lib/user.rb

```diff
@@ -0,0 +1,5 @@
+class User
+
```

Comment:
Is there room here for a username as well?  Or is name and username/handle the same?